server/src/main/webapp/WEB-INF/rails/webpack/views/pages/elastic_agent_configurations/cluster_profile_widget.tsx

```diff
@@ -65,8 +65,8 @@ interface HeaderAttrs {
   image: m.Children;
 }
 
-class ClusterProfileHeaderWidget extends MithrilComponent<HeaderAttrs> {
-  view(vnode: m.Vnode<HeaderAttrs, State>) {
+class ClusterProfileHeaderWidget extends MithrilViewComponent<HeaderAttrs> {
+  view(vnode: m.Vnode<HeaderAttrs>) {
```

Comment:
Ahh. I think this is the bit I got stuck on and didn't know how to fix due to my gross lack of knowledge on both Mithril and Typescript 😓

Reply:
:D 
We've introduced `MithrilComponent` for a component with state and `MithrilViewComponent` for a stateless component.
`ClusterProfileHeaderWidget` component was wrongly implemented as `MithrilComponent`, where it does not have any state.